src/Application/DegreeProgramBulkUpdater.php

```diff
@@ -19,10 +20,21 @@ public function __construct(
     }
 
     /**
-     * Degree program values are calculated based on the degree term.
-     * So if the term was changed, we have to update all related degree programs.
+     * Degree program slug is calculated based on the degree term meta abbreviation value.
+     * So if the abbreviation value was changed, we have to update all related degree programs.
```

Comment:
Let's take the opportunity to improve the comment wording according to the Boy Scout rules.
```suggestion
     * Degree program slugs are calculated based on the "abbreviation" term meta value.
     * Therefore, if the abbreviation changes, we must update all related programs.
```